js2/js_hw2_t3.js

```diff
@@ -0,0 +1,20 @@
+const minAge = 18;
+const maxAge = 60;
+//let age = "10"; //Вывело You don't have access cause your age is 10 It's less then 18
+let age = "ten"; //Вывело This is not a number. Enter age as a number
+if(isNaN(age)) {
+    let age = typeof Number
+    console.log("This is not a number. Enter age as a number");  
+} else {
+    if (age < minAge) {
```

Comment:
Не совсем понятна даная строка `let age = typeof Number`, зачем она тебе ?

После проверки можно `age` привести к числу, так оно не будет приводиться к числу в каждом из `if` ветвлений 
```suggestion
let age = "ten"; //Вывело This is not a number. Enter age as a number
if(isNaN(age)) {
    console.log("This is not a number. Enter age as a number");  
} else {
   age = +age; // Number(age) 
    if (age < minAge) {
```

Reply:
это для того, чтоб строке присвоился тип number если он изначально NaN

Reply:
у NaN тип Number в любом случае